app/views/posts/index.html.erb

```diff
@@ -1 +1,43 @@
-<h1>Here is the list of all posts for user A </h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <div class='container'>
+      <div class='card'>
+        <img class='user_pic' src= <%= @user.photo %> >
+        <div class='left_card'>
+          <h2 class='username'><%= @user.name %></h2>
+          <p class='count'>Number of posts: <%= @user.posts_counter >= 0 ? @user.posts_counter : '0' %></p><br>
+        </div>
+      </div>
+      <div class='bio_div'>
+        <p>Bio</p>
+        <p><%= @user.bio %></p>
+      </div>
+
+        <% @posts.each do |post| %>
+          <a href=<%= user_post_url(@user.id, post.id) %> class='sub_container'>
+            <div class='postcard'>
+              <p><%= post.title %></p>
+              <p><%= post.text.truncate(50, separator:' ') %></p>
+              <p class='count'>
+                Comments: <%= post.comments_counter != nil ? post.comments_counter : '0' %>
+                Likes: <%= post.likes_counter != nil ? post.likes_counter : '0' %>
+              </p>
+            </div>
+            <% @comments = post.recent_comments %>
+            <% if @comments.length > 0 %>
+              <div class='comments_card'>
+                <% @comments.each do |comment| %>
+                  <% user = User.find(comment.author_id)%>
+                  <p class='comment'><%= user.name %>: <%= comment.text %></p>
+                <% end %>
+              </div>
+            <% end %>
+          </a>
+        <% end %>
```

Comment:
- [ ] You should pay more attention to the function as styling is nice to have the **See all Post** links open the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) However your design does not match the wireframe👇🏻 Kindly make sure that the index view page matches the given  [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/231671728-a1337dab-99e0-4583-9bb5-8e1ec1590608.png) | ![image](https://user-images.githubusercontent.com/79058364/231671531-0b0de30c-f840-4c97-a525-15ae6cc6da35.png)

- [ ]  Use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page. eg:

   ```ruby
    <% post.recent_comments.each do |comment| %>
     <p><%= comment.author.name %>: <%= comment.text %></p> 
    <% end %>
   ```

Reply:
Okay bro thanks to give me this suggestion, but as you can see below I have added the same implementation initially, I think it's an issue with my seeds. rb file it's missing a comment but I have updated it with a new seed file you can look at it.
![image](https://user-images.githubusercontent.com/104406349/231864431-87a9854e-c1c4-4069-839a-ead8169fcd66.png)
 But all functionality is working:
- 1 root(users#index)
![image](https://user-images.githubusercontent.com/104406349/231865024-72adf28b-e6eb-4bfc-b1f7-580307db54cb.png)

2 users#show
![image](https://user-images.githubusercontent.com/104406349/231865223-0be49709-cb5a-4f43-b565-3ab54373d2e9.png)

3 posts# index
![image](https://user-images.githubusercontent.com/104406349/231865398-468ad9b6-f126-4a71-81f5-cd4239ec5328.png)

4 posts#show
![image](https://user-images.githubusercontent.com/104406349/231865518-0959ba85-e2dd-4d25-95f6-656e1998cff4.png)